demos/README.md

```diff
@@ -35,7 +35,7 @@ $ sudo buildah login quay.io
 $ buildah login quay.io
 ```
 
-There are several variables you will want to set that are listed at the top of the script. The name for the container image, your quay.io username, your name, and the Fedora release number:
+There are several variables you will want to set that are listed at the top of the script. The name for the container image, your Quay.io username, your name, and the Fedora release number:
```

Comment:
quay.io is a web site, it should not be capitalized?

Reply:
This was my rationale @TomSweeneyRedHat, I was originally going to lowercase the references because they were mixed, but checked how it was displayed on the site. Cheers for checking 😄